### ISSUE TITLE
Inotify.__handle_inotify_event raises another TypeError when attempting to handle an event in python3.

### ISSUE THREAD

**Issue by declaresub** [dsoprea/PyInotify#7]
Having fixed the two problems recently reported, another TypeError is raised while handling an event.

Traceback (most recent call last):
  File "/inotify/dev/test.py", line 50, in <module>
    _main()
  File "/inotify/dev/test.py", line 38, in _main
    for event in i.event_gen():
  File "/inotify/inotify/adapters.py", line 162, in event_gen
    in self.__handle_inotify_event(fd, event_type):
  File "/inotify/inotify/adapters.py", line 141, in __handle_inotify_event
    filename = filename.rstrip('\0')
TypeError: 'str' does not support the buffer interface

Here, the problem is that filename is a 'bytes' object in python 3, so needs to be decoded before stripping.  My fix was this:

```
filename = filename.encode(sys.getdefaultencoding()).rstrip('\0')
```

**Comment by dsoprea** [comment]
If you submit it in a PR you'll get the credit :) . I realize it's a small
fix but I'm pretty saturated with other projects tonight.



Dustin

**Comment by declaresub** [comment]
I can do that.

**Comment by dsoprea** [comment]
Bump.

**Comment by declaresub** [comment]
You have my pull request...

**Comment by sergio91pt** [comment]
Fixed in 74d182112332269d436b8381c02468f95fa87ca3
filename is now a `bytes`
